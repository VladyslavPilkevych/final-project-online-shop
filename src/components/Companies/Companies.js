import React from 'react';
import style from './Companies.module.scss';
import imageRoccat from '../../assets/images/companies-images/image-roccat.png';
import imageMsi from '../../assets/images/companies-images/image-msi.png';
import imageRazer from '../../assets/images/companies-images/image-razer.png';
import imageThermaltake from '../../assets/images/companies-images/image-thermaltake.png';
import imageAdata from '../../assets/images/companies-images/image-adata.png';
import imageHp from '../../assets/images/companies-images/image-hp.png';
import imageGigabyte from '../../assets/images/companies-images/image-gigabyte.png';

function Companies() {
  return (
    <div className={style.companiesLogos}>
      <img src={imageRoccat} className={style.companieImage} alt="logo-roccat" />
      <img src={imageMsi} className={style.companieImage} alt="logo-msi" />
      <img src={imageRazer} className={style.companieImage} alt="logo-razer" />
      <img src={imageThermaltake} className={style.companieImage} alt="logo-thermaltake" />
      <img src={imageAdata} className={style.companieImage} alt="logo-adata" />
      <img src={imageHp} className={style.companieImage} alt="logo-hp" />
      <img src={imageGigabyte} className={style.companieImage} alt="logo-gigabyte" />
    </div>
  );
}

export default Companies;
