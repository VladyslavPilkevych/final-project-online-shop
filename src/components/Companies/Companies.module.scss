.companiesLogos {
    max-width: 1399px;
    margin: 0 auto;
    display: grid;
    grid-template-columns: repeat(7,1fr);
    justify-self: center;
    margin-top: 30px;
    margin-bottom: 42px;
}

.companieImage {
    max-width: 200px;
}

@media screen and (min-device-width: 1204px) and (max-device-width: 1397px) {
    .companiesLogos {
       display: grid;
       justify-items: center;
       grid-template-columns: repeat(5, 1fr);
    }
}

@media screen and (min-device-width: 769px) and (max-device-width: 1203px) {
    .companiesLogos {
       display: grid;
       grid-template-columns: repeat(3, 1fr);
       align-items: center;
       justify-items: center;
    }
}

@media screen and (min-device-width: 100px) and (max-device-width: 768px) {
    .companiesLogos {
       display: grid;
       grid-template-columns: repeat(2, 1fr);
       align-items: center;
       justify-items: center;
    }

    .companieImage {
        max-width: 170px;
    }
}

@media screen and (min-device-width: 100px) and (max-device-width: 340px) {
    .companieImage {
        max-width: 120px;
    }
}
    
