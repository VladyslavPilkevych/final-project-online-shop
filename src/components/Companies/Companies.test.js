import { render } from '@testing-library/react';
import Companies from './Companies';

describe('Snapshot companies should render', () => {
  test('Snapshot companies is rendered', () => {
    const { asFragment } = render(<Companies />);
    expect(asFragment()).toMatchSnapshot();
  });
});
