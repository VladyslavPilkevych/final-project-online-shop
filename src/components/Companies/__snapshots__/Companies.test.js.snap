// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot companies should render Snapshot companies is rendered 1`] = `
<DocumentFragment>
  <div
    class="companiesLogos"
  >
    <img
      alt="logo-roccat"
      class="companieImage"
      src="image-roccat.png"
    />
    <img
      alt="logo-msi"
      class="companieImage"
      src="image-msi.png"
    />
    <img
      alt="logo-razer"
      class="companieImage"
      src="image-razer.png"
    />
    <img
      alt="logo-thermaltake"
      class="companieImage"
      src="image-thermaltake.png"
    />
    <img
      alt="logo-adata"
      class="companieImage"
      src="image-adata.png"
    />
    <img
      alt="logo-hp"
      class="companieImage"
      src="image-hp.png"
    />
    <img
      alt="logo-gigabyte"
      class="companieImage"
      src="image-gigabyte.png"
    />
  </div>
</DocumentFragment>
`;
